Content.Server/Atmos/Components/GasAnalyzerComponent.cs

```diff
@@ -263,24 +257,12 @@ async Task<bool> IAfterInteract.AfterInteract(AfterInteractEventArgs eventArgs)
             return true;
         }
 
-
-
         void IDropped.Dropped(DroppedEventArgs eventArgs)
         {
             if (_entities.TryGetComponent(eventArgs.User, out ActorComponent? actor))
             {
                 CloseInterface(actor.PlayerSession);
             }
         }
```

Comment:
You can also just delete IDropped from this component.

IMO its fine to use the analyzer when its not in your hands, but if it really matters the activatable UI component has a `InHandsOnly` field.

Reply:
actually I'll just do that in a separate PR, because then only the gas tank internals use it, and IDropped can just be removed altogether